src/__tests__/test-uuid.js

```diff
@@ -5,12 +5,31 @@ describe('uuid', () => {
         expect(_UUID('v7')).toHaveLength(36)
     })
 
-    it('generates many unique ids in a reasonable time', () => {
+    it('generates many unique v7 UUIDs in a reasonable time', () => {
         const ids = Array.from({ length: 500_000 }, () => _UUID('v7'))
         expect(new Set(ids).size).toBe(ids.length)
     })
 
+    it('generates many unique OG UUIDs in a reasonable time', () => {
+        const ids = Array.from({ length: 500_000 }, () => _UUID())
+        expect(new Set(ids).size).toBe(ids.length)
+    })
+
     it('by default should be the format we have used forever', () => {
         expect(_UUID().length).toBeGreaterThanOrEqual(52)
     })
+
+    it('generates different UUIDs when window.performance is available', () => {
+        const uuids = Array.from({ length: 1000 }, () => _UUID())
+
+        expect(new Set(uuids).size).toBe(uuids.length)
```

Comment:
Could the [test above](https://github.com/PostHog/posthog-js/pull/727/files#diff-32d68a97f6839986112d39f2b3cd933dd8052b6cf8cee8f64510473c6987434bR13-R16) be consolidated in here (or this assertion removed) given it is already tested above